extensions/wikia/ImageReview/ImageListGetter.class.php

```diff
@@ -315,4 +318,54 @@ private function getInReviewState( int $state ) : int {
 
 		else return ImageReviewStates::IN_REVIEW;
 	}
+
+	private function getLabelValues() {
+		$labels = [];
+
+		$labels['labelOk'] = [
+			'value' => ImageReviewStates::APPROVED,
+			'checked' => ''
+		];
+
+		$labels['labelDelete'] = [
+			'value' => $this->state == 'rejected' ? ImageReviewStates::DELETED : ImageReviewStates::REJECTED,
+			'checked' => ''
+		];
+
+		$labels['labelQuestionable'] = [
+			'value' => ImageReviewStates::QUESTIONABLE,
+			'checked' => ''
+		];
+
+		return $labels;
+	}
+
+	/**
+	* Add additional markup for various flagged states
+	* @param integer $
+	* @return object
+	*/
+	private function getFlagsContent( $flag ) {
+		$flagsContent = [];
+
+		if( $flag & ImageListGetter::FLAG_SUSPICOUS_USER ) {
+			$flagsContent['flag_suspicous_user'] = [
+				'title' => "Flagged: Susicious user. Click to go to uploader's profile"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Flagged: Susicious user. Click to go to uploader's profile" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)